BGAnimations/ScreenEvaluation common/Shared/RpgOverlay.lua

```diff
@@ -1,13 +1,37 @@
 local NumEntries = 13
 local RowHeight = 24
 local RpgYellow = color("0.902,0.765,0.529,1")
+local ItlPink = color("1,0.2,0.406,1")
 
 local paneWidth1Player = 330
 local paneWidth2Player = 230
 local paneWidth = (GAMESTATE:GetNumSidesJoined() == 1) and paneWidth1Player or paneWidth2Player
 local paneHeight = 360
 local borderWidth = 2
 
+local SetRpgStyle = function(eventAf)
+	eventAf:GetChild("MainBorder"):diffuse(RpgYellow)
+	eventAf:GetChild("BackgroundImage"):visible(true)
+	eventAf:GetChild("BackgroundColor"):diffuse(color("0,0,0,0.3"))
+	eventAf:GetChild("HeaderBorder"):diffuse(RpgYellow)
+	eventAf:GetChild("HeaderBackground"):diffusetopedge(color("0.275,0.510,0.298,1")):diffusebottomedge(color("0.235,0.345,0.184,1"))
+	eventAf:GetChild("Header"):diffuse(RpgYellow)
+	eventAf:GetChild("EX"):diffuse(RpgYellow):visible(false)
```

Comment:
I guess there is no diffuse necessary when making it invisible.

Reply:
Removed the diffuse